<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="VisaoXY" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>856</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonNew">
       <property name="text">
        <string>NEW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutVoxel">
       <property name="text">
        <string>PutVoxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutVoxel">
       <property name="text">
        <string>CutVoxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutBox">
       <property name="text">
        <string>PutBox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutBox">
       <property name="text">
        <string>CutBox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutEllipsoid">
       <property name="text">
        <string>PutEllipsoid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutEllipsoid">
       <property name="text">
        <string>CutEllipsoid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutSphere">
       <property name="text">
        <string>PutSphere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutSphere">
       <property name="text">
        <string>CutSphere</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxRaio">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="title">
        <string>Raio </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>251</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QLCDNumber" name="lcdNumberRaio"/>
         </item>
         <item row="0" column="2">
          <widget class="QSlider" name="horizontalSliderRaio">
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxElipse">
       <property name="title">
        <string>Elipse</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelRx">
            <property name="text">
             <string>   Rx  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelRy">
            <property name="text">
             <string> Ry</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelRz">
            <property name="text">
             <string>Rz</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderRX">
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderRZ">
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSliderRY">
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber_4"/>
          </item>
          <item row="1" column="2">
           <widget class="QLCDNumber" name="lcdNumber_5"/>
          </item>
          <item row="2" column="2">
           <widget class="QLCDNumber" name="lcdNumber_6"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCaixa">
       <property name="title">
        <string>Caixa</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QLCDNumber" name="lcdNumber_3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDimY">
            <property name="text">
             <string>DimY</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelDimZ">
            <property name="text">
             <string>DimZ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderDimZ">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLCDNumber" name="lcdNumber_2"/>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSliderDimY">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDimx">
            <property name="text">
             <string>DimX</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderDimX">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>781</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxCoordenadas">
       <property name="title">
        <string>Profundidade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>22</y>
          <width>341</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderZ">
           <property name="maximum">
            <number>29</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelZ">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLCDNumber" name="lcdNumberZ"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Cor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>351</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSliderG">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLCDNumber" name="lcdNumberB"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelB">
           <property name="text">
            <string>B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="lcdNumberG"/>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSliderB">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelG">
           <property name="text">
            <string>G</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLCDNumber" name="lcdNumberR"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelR">
           <property name="text">
            <string>R</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderR">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionsave"/>
   <addaction name="actionPutV"/>
   <addaction name="actionCutV"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutB"/>
   <addaction name="actionPutE"/>
   <addaction name="actionCutE"/>
   <addaction name="actionPutEs"/>
   <addaction name="actionCutEs"/>
  </widget>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/voxel.png</normaloff>:/icones/recursos/voxel.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/save.png</normaloff>:/icones/recursos/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/new.png</normaloff>:/icones/recursos/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionPutV">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/voxel.png</normaloff>:/icones/recursos/voxel.png</iconset>
   </property>
   <property name="text">
    <string>PutV</string>
   </property>
  </action>
  <action name="actionCutV">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cutvoxel.png</normaloff>:/icones/recursos/cutvoxel.png</iconset>
   </property>
   <property name="text">
    <string>CutV</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/putbox.png</normaloff>:/icones/recursos/putbox.png</iconset>
   </property>
   <property name="text">
    <string>PutBox</string>
   </property>
  </action>
  <action name="actionCutB">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cutbox.png</normaloff>:/icones/recursos/cutbox.png</iconset>
   </property>
   <property name="text">
    <string>CutB</string>
   </property>
  </action>
  <action name="actionPutE">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/putEllipse.png</normaloff>:/icones/recursos/putEllipse.png</iconset>
   </property>
   <property name="text">
    <string>PutE</string>
   </property>
  </action>
  <action name="actionCutE">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cutEllipse.png</normaloff>:/icones/recursos/cutEllipse.png</iconset>
   </property>
   <property name="text">
    <string>CutE</string>
   </property>
  </action>
  <action name="actionPutEs">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/putcircle.png</normaloff>:/icones/recursos/putcircle.png</iconset>
   </property>
   <property name="text">
    <string>PutEs</string>
   </property>
  </action>
  <action name="actionCutEs">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/recursos/cutcircle.png</normaloff>:/icones/recursos/cutcircle.png</iconset>
   </property>
   <property name="text">
    <string>CutEs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VisaoXY</class>
   <extends>QWidget</extends>
   <header>visaoxy.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRaio</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberR</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberG</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberB</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>sair()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sair()</slot>
 </slots>
</ui>
